common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -12,9 +12,12 @@ class GradesheetTest(unittest.TestCase):
 
     def test_weighted_grading(self):
         scores = []
-        agg_fields = dict(display_name="aggregated_score", module_id=None)
-        prob_fields = dict(display_name="problem_score", module_id=None, raw_earned=0, raw_possible=0, weight=0)
+        agg_fields = dict(display_name="aggregated_score", module_id=None, attempted=False)
+        prob_fields = dict(
+            display_name="problem_score", module_id=None, raw_earned=0, raw_possible=0, weight=0, attempted=False,
```

Comment:
Don't we usually go to one line per item once a parameter list goes over the length of a single line?